brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
👍🏻

Sent from my iPhone

> On Nov 26, 2015, at 2:23 PM, Michał Muskała <notifications@github.com> wrote:
> 
> In brunch-config.js:
> 
> > @@ -31,8 +31,7 @@ exports.config = {
> >    plugins: {
> >      babel: {
> >        // Do not use ES6 compiler in vendor code
> > -      ignore: [/^(web\/static\/vendor)/],
> > -      loose: "all"
> Microsoft is soon dropping the support for old IEs, so it may be one of the reasons to drop it here too
> 
> —
> Reply to this email directly or view it on GitHub.
>